packages/spacecat-shared-data-access/src/models/importer/import-url.js

```diff
@@ -12,7 +12,13 @@
 
 import { isValidUrl } from '@adobe/spacecat-shared-utils';
 import { Base } from '../base.js';
-import { IMPORT_JOB_STATUS } from './import-job.js';
+
+export const IMPORT_URL_STATUS = {
```

Comment:
see comment here https://github.com/adobe/spacecat-api-service/pull/341/files#r1632614363

Reply:
Good call @solaris007, this is much tidier on the API side:

```
import { ImportJobStatus } from '@adobe/spacecat-shared-data-access';
```